news/2023/2023-07-22-featured-artist-track-updates-summer-2023.md

```diff
@@ -0,0 +1,427 @@
+- [**Pratanallis**](#pratanallis)
+- [**Riya**](#riya)
+- [**seatrus**](#seatrus)
+- [**seleP**](#selep)
+- [**solfa**](#solfa)
+- [**Trial & Error**](#trial-&-error)
+- [**Wisp X**](#wisp-x)
+
+Songs marked with this fancy *NEW* icon are... new. (Hint: you should map these songs.)
+
+![](/wiki/shared/news/2022-01-02-featured-artist-track-updates-winter-2022/new.png)
+
+New songs are also listed below along with any relevant Ranked/Loved maps and an audio preview or two. These should give you a quick impression of what's in store, but also remember that all song previews are available from each Featured Artist page!
+
+---
+
+## [BilliumMoto and niu arx / Dictate](https://osu.ppy.sh/beatmaps/artists/38)
+
+[![](https://assets.ppy.sh/artists/38/header.jpg)](https://osu.ppy.sh/beatmaps/artists/38)
+
+Original songs for tournament tiebreakers are everywhere these days, but we doubt you've seen a tournament like [Jaye's One Key Event 2](https://osu.ppy.sh/community/forums/topics/1430912?n=1) — an osu! mania tournament where every map has... one key. It's ridiculous, and both [**BilliumMoto**](https://osu.ppy.sh/beatmaps/artists/38) and [**niu arx**](https://osu.ppy.sh/beatmaps/artists/21) (known as *Dictate* at the time) made the equally ridiculous song [**1KARUGA**](https://osu.ppy.sh/beatmapsets/1649302) for its finale.
+
+Oh, the song was also used as [the tiebreaker for TWC 2023's Finals mappool](https://osu.ppy.sh/beatmapsets/1976834). So that's cool.
+
+### BilliumMoto & Dictate - 1KARUGA
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/38/Songs/BilliumMoto%20%26%20Dictate%20-%201KARUGA.mp3" type="audio/mpeg">
+</audio>
+
+## [ELFENSJoN](https://osu.ppy.sh/beatmaps/artists/69)
+
+[![](https://assets.ppy.sh/artists/69/header.jpg)](https://osu.ppy.sh/beatmaps/artists/69)
+
+These **3** new songs by **ELFENSJoN** have the all the essentials of osu! music: high-paced gothic J-rock and the vocal mastery of veteran utaite *Hanatan*.
+
+If that's not convincing already, you may need to reconsider your preferences.
+
+### ELFENSJoN - DAWN
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/69/DAWN/ELFENSJoN%20-%20DAWN.mp3" type="audio/mpeg">
+</audio>
+
+### ELFENSJoN - ECLIPSE
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/69/ECLIPSE/ELFENSJoN%20-%20ECLPISE.mp3" type="audio/mpeg">
+</audio>
+
+### ELFENSJoN - Magatsu Yami ni Utau
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/69/Magatsu%20Yami%20ni%20Utau%2FELFENSJoN%20-%20Magatsu%20Yami%20ni%20Utau.mp3" type="audio/mpeg">
+</audio>
+
+## [Good Kid](https://osu.ppy.sh/beatmaps/artists/240)
+
+[![](https://assets.ppy.sh/artists/240/header.jpg)](https://osu.ppy.sh/beatmaps/artists/240)
+
+**Good Kid**'s third EP of hype-filled upbeat rock was released earlier this year (aptly titiled *Good Kid 3*). Since all of their already-licensed songs are already mapped, it only made sense for us to snag **4** of their new tunes.
```

Comment:
```suggestion
**Good Kid**'s third EP of hype-filled upbeat rock was released earlier this year (aptly titled *Good Kid 3*). Since all of their already licensed songs are already mapped, it only made sense for us to snag **4** of their new tunes.
```